scripts/install_python.sh

```diff
@@ -22,14 +22,22 @@ apt_install \
     python3-venv \
     swig
 
+# Setup a virtualenv to install things into
+
+# Put things under /opt/venv, if nothing else is specified
+export VIRTUAL_ENV="${VIRTUAL_ENV:=/opt/venv}"
+export PATH="${VIRTUAL_ENV}/bin:${PATH}"
+
+# Make sure that Rstudio sees these env vars too
+echo "PATH=${PATH}" >>"${R_HOME}/etc/Renviron.site"
```

Comment:
@eitsupi did we have some logic elsewhere to avoid repeatedly writing new definitions of PATH to `Renviron.site` ?  It's probably fine but if this script is run repeatedly it can make that file a bit clunky, and I thought I recalled (or imagined?) you having some nice trick to keep it tidy?